api/v1alpha1/openapi.yaml

```diff
@@ -940,6 +949,31 @@ components:
           type: string
           description: The active state of the systemd unit.
       description: The status of the systemd unit.
+    EnrollmentRequestApproval:
+      type: object
+      properties:
+        fleetName:
```

Comment:
Is the enrollment per fleet?  If I want to move the device to another fleet, I need a new enrollment?

Reply:
We could later on change the fleetName field of a device, if we want to allow that.

Reply:
We're enrolling devices into the system. Reassigning to another fleet shouldn't require a new enrolment, nor should migrating to another service instance in another region.

Reply:
We could remove the fleetName from the approval, so then the enrollment approval stops being related to a specific fleet.